kafka-connect-jms/src/main/scala/com/datamountaineer/streamreactor/connect/jms/JMSSessionProvider.scala

```diff
@@ -45,29 +45,11 @@ case class JMSSessionProvider(queueConsumers: Map[String, MessageConsumer],
     connection.start()
   }
 
-  def close(): Try[Unit] = {
```

Comment:
does it release the consumers/producers automatically when connection/context is closed?

Reply:
I tried closing the comsumers/producers after closing the connection and it said they were already closed 👍